transit_odp/timetables/dataframes.py

```diff
@@ -157,31 +157,113 @@ def stop_point_refs_to_dataframe(stop_point_refs):
 def provisional_stops_to_dataframe(stops, system=None):
     stop_points = []
     for stop in stops:
+        flx_zone_locations = []
         atco_code = stop.get_element(["AtcoCode"]).text
-        location = stop.get_element(["Place", "Location"])
-
-        if system is None or system.lower() == GRID_LOCATION.lower():
-            easting = location.get_element(["Translation", "Easting"]).text
-            northing = location.get_element(["Translation", "Northing"]).text
-            geometry = (
-                grid_gemotry_from_str(easting, northing)
-                if easting and northing
-                else None
-            )
-
-        elif system.lower() == WSG84_LOCATION.lower():
-            latitude = location.get_element(["Translation", "Latitude"]).text
-            longitude = location.get_element(["Translation", "Longitude"]).text
-            geometry = (
-                wsg84_from_str(longitude, latitude) if latitude and longitude else None
-            )
-        locality_id = stop.get_element(["Place", "NptgLocalityRef"]).text
-        stop_points.append(
-            {"atco_code": atco_code, "geometry": geometry, "locality": locality_id}
+        stop_classification = stop.get_element(
+            ["StopClassification", "OnStreet", "Bus"]
```

Comment:
could we confirm if `StopClassification` element is present then all the child elements listed here will also be present and are mandatory elements? If not, null checks will be required

Reply:
added the check

Reply:
Would this return None if OnStreet element is not present?